pages/common/quarkus.md

```diff
@@ -0,0 +1,36 @@
+# quarkus
+
+> Create projects, manage extensions and perform essential build and development tasks.
+> More information: <https://quarkus.io/guides/cli-tooling>.
+
+- Create a new application project:
```

Comment:
```suggestion
- Create a new application project:
```

Does this create a new directory? Or the project is initialized on the same directory? (non-blocking change, i.e. we can merge this PR and clarify this later)

Reply:
Yes, it does